packages/react-async/documentation/recipes.md

```diff
@@ -5,6 +5,8 @@ This document covers a few common scenarios where `@shopify/react-async` and rel
 ## Table of contents
 
 1. [Index pages with empty states](#index-pages-with-empty-states)
+1. [UI that is initially hidden](#ui-that-is-initially-hidden)
+1. [Below-the-fold content](#below-the-fold-content)
```

Comment:
You can increment numbers.

Reply:
Markdown auto-increments them, I prefer this in case I rearrange/ add more/ remove some.

Reply:
TIL